app/routes.py

```diff
@@ -110,26 +113,48 @@ def categoryHandler(id):
         except Exception as e:
             return str(e), 500
 
-
-    else: 
+    else:
         if (Categories.findById(id) is None):
             return 'fail', 204
         Categories.delete(id)
         return "success", 200
 
 ################################################################################
-#matches 
+# matches
 ################################################################################
-    
-@cross_origin()
-@app.route("/api/matches", methods = ["GET"])
-def getMatches():
-    app.logger.debug('This endpoint was hit!')
-    matches = Matches.query.all()
-    return json.dumps([m.serialize() for m in matches])
 
-@cross_origin()
-@app.route("/api/match/<id>", methods = ["DELETE", "GET", "PUT"])
+
+# @cross_origin()
+# @app.route("/api/matches", methods=["GET"])
```

Comment:
just realized that we still might need this for the admin page, and for andrew, maybe we can move the old get all matches to something like api/matches/all for now, and later we can all the all functionality using request params